orbit/cmd/orbit/orbit.go

```diff
@@ -616,6 +616,31 @@ func main() {
 			Platform:       osqueryHostInfo.Platform,
 		}
 
+		// Get the hardware UUID. We use a temporary RocksDB location in order to guarantee that
+		// we're getting true UUID, not a cached UUID. See
+		// https://github.com/fleetdm/fleet/issues/17934 and
+		// https://github.com/osquery/osquery/issues/7509 for more details.
+
+		oi, err := getHostInfo(osquerydPath, filepath.Join(os.TempDir(), "tmp-db"))
+		if err != nil {
+			return fmt.Errorf("get UUID from temp db: %w", err)
+		}
+
```

Comment:
`RemoveAll` `filepath.Join(os.TempDir(), "tmp-db")` after it was used?

Reply:
I definitely had this in a `defer` call and I think I removed it by mistake 😅